cirq/ops/pauli_string.py

```diff
@@ -1024,3 +1044,49 @@ def _decompose_into_cliffords(op: 'cirq.Operation') -> List['cirq.Operation']:
 
     raise TypeError(f'Operation is not a known Clifford and did not decompose '
                     f'into known Cliffords: {op!r}')
+
+
+# Mypy has extreme difficulty with these constants for some reason.
+_i = cast(identity.IdentityGate, identity.I)  # type: ignore
+_x = cast(pauli_gates.Pauli, pauli_gates.X)  # type: ignore
+_y = cast(pauli_gates.Pauli, pauli_gates.Y)  # type: ignore
+_z = cast(pauli_gates.Pauli, pauli_gates.Z)  # type: ignore
+
+PAULI_GATE_LIKE_TO_GATE_MAP: Dict['cirq.PAULI_GATE_LIKE',
+                                  Union['cirq.Pauli', 'cirq.IdentityGate']] = {
```

Comment:
weird line wrapping

Reply:
Yeah. yapf wants what it wants, who are we to judge.